azure-pipelines-template.yml

```diff
@@ -0,0 +1,24 @@
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  strategy:
+    matrix:
+      node_10_x:
+        node_version: 10.x
+      node_9_x:
+        node_version: 9.x
+      node_8_x:
+        node_version: 8.x
+      node_6_x:
+        node_version: 6.x
+    maxParallel: 4
+  steps:
+  - task: NodeTool@0
+    inputs:
+      versionSpec: $(node_version)
+    displayName: 'Install Node.js'
+  - bash: npm i
+    displayName: 'Install dependencies'
+  - bash: npm run coveralls -- -c
```

Comment:
@StarpTech `npm run coveralls -- -c` does not run linting & typescripts checks

Can you change it to `npm run lint && npm run coveralls -- -c && npm run typescript`

Reply:
or maybe just change `package.json` script to include all?

Reply:
Good catch. I added them.